src/Test/TestSaveLoad.java

```diff
@@ -62,7 +62,7 @@ private boolean testSaveLoad()
     String path = game.writeSave(false);
     File file = new File(path);
     testPassed &= validate(file.exists(),  "    The file we just saved to doesn't exist");
-    testPassed &= validate(GameInstance.isSaveCompatible(path).length() == 0,  "    We are incompatible with the save we just made");
+    testPassed &= validate(GameInstance.getSaveWarnings(path).length() == 0,  "    We are incompatible with the save we just made");
```

Comment:
Idiomatic

Reply:
The length of the warnings is zero. Perfectly idiomatic.